examples/fortran/default.f90

```diff
@@ -0,0 +1,7 @@
+! Type your code here, or load an example.
```

Comment:
Gosh, I've not seen Fortran code since my university days, in the 1990s! :)

Reply:
Well, a nicer example would have been good then (I went with the spirit of the other very short snippets). But isn't it good to see anyway? :)